crates/holochain_types/src/wasmer_types.rs

```diff
@@ -47,16 +47,12 @@ pub fn build_ios_module(wasm: &[u8]) -> Result<Module, CompileError> {
         "Found wasm and was instructed to serialize it for ios in wasmer format, doing so now..."
     );
     let compiler_config = cranelift();
-    let ios_target = wasmer_ios_target();
-    let engine = Dylib::new(compiler_config).target(ios_target).engine();
-    let store = Store::new(&engine);
+    let store = Store::new(compiler_config);
     Module::from_binary(&store, wasm)
 }
 
 /// Generate a headless Dylib Store suitable for iOS.
 /// Useful for re-building an iOS Module from a preserialized WASM Module.
 pub fn ios_dylib_headless_store() -> Store {
-    let ios_target = wasmer_ios_target();
-    let engine = Dylib::headless().target(ios_target).engine();
-    Store::new(&engine)
+    Store::default()
```

Comment:
hi @thedavidmeister 
as an experimental contribution to holochain I didn't manage to put this under test coverage, but I'm having doubts that this would work in practice. Based on changes to wasmer was it necessary to get rid of the Dylib usage and ios target references here?

Reply:
There are actually new movements within tx5 that make native iOS possible now so I don't want to backslide

Reply:
@Connoropolous dylib was removed in wasmer 3.x i think, or if it's still there it's not at all how it used to be, the feature was removed and engines no longer exist

https://github.com/wasmerio/wasmer/blob/master/docs/migration_to_3.0.0.md#rationale-for-changes-in-300

> The Engines API has been simplified, Instead of the user choosing and setting up an engine explicitly, everything now uses a single engine. All functionalities of the universal, staticlib and dylib engines should be available in this new engine unless explicitly stated as unsupported.

Reply:
Sounds like it might/should work. Hoping so.